pkg/identityserver/store/api_key_store.go

```
@@ -132,3 +132,18 @@ func (s *apiKeyStore) UpdateAPIKey(ctx context.Context, entityID ttnpb.Identifie
 	}
 	return keyModel.toPB(), nil
 }
+
+func (s *apiKeyStore) DeleteEntityAPIKeys(ctx context.Context, entityID ttnpb.Identifiers) error {
+	defer trace.StartRegion(ctx, "delete entity api keys").End()
+	entity, err := s.findDeletedEntity(ctx, entityID, "id")
+	if err != nil {
+		return err
+	}
+	var keyModels []APIKey
+	query := s.query(ctx, APIKey{}).Where(&APIKey{EntityID: entity.PrimaryKey(), EntityType: entityTypeForID(entityID)})
+	err = query.Find(&keyModels).Error
+	if err != nil {
+		return err
+	}
```

I don't think this is necessary. We don't need to return any errors if no API keys found.